2015-10-27.md

```diff
@@ -0,0 +1,32 @@
+Hi Tesselators!
+
+It's been a few weeks since the last This Week in Tessel, our apologies! We've been hard at work on the Tessel repos, building and growing the project.
```

Comment:
You can remove this entirely, no reason to make this excuse / apology for not sending out announcements unless you're promising to keep on top of it better (which would be a difficult promise to make and keep =/ )